wkt/src/__init__.py

```diff
@@ -0,0 +1,27 @@
+"""Module with WKT wrapper for shapely"""
+
+from wkt.src.geometries import (  # noqa
+    WKTObject,
+    PointModel,
+    LineStringModel,
+    PolygonModel,
+    MultiPointModel,
+    MultiLineStringModel,
+    MultiPolygonModel,
+    GeometryType,
+)
+from wkt.src.parsing import (  # noqa
```

Comment:
why noqa?

Reply:
Because otherwise flake8 complains about unused import.

Reply:
nevermind, looks like it doesn't complain anymore (probably just was left over from before). removed # noqa